CDP4ShellDialogs/ViewModels/ModelOpeningDialogViewModel.cs

```diff
@@ -351,6 +351,30 @@ private void PopulateSessionsRowViewModel(IEnumerable<ISession> sessions)
             }
 
             this.SelectedRowSession = this.SessionsAvailable.FirstOrDefault();
+
+            var modelSelectionSessionRowViewModel = this.SelectedRowSession;
+
+            if (modelSelectionSessionRowViewModel != null)
+            {
+                this.SortDomainOfExpertise();
```

Comment:
the solution is far more simple i think:

have a look at the `ModelSelectionIterationSetupRowViewModel` class on line 50:

```
this.DomainOfExpertises.AddRange(participant.Domain);
```

if you do:

```
this.DomainOfExpertises.AddRange(participant.Domain.OrderBy(x => x.ShortName));
```

does that not solve it at the root? in your solution the row-view-models are created, properties are set, and then later you change them (clear the `DomainOfExpertises` to then later repopulate?